### ISSUE TITLE
Face parameters 

### ISSUE THREAD

**Issue by waquey** [HiPer0/HiPer#1]
Dear great authors, 

Thanks for the great works. I've tried to generate dogs and it works perfectly. 
However, when I try to generate face, the identity does not keep. 

Should I change the parameter instead? (I've tried to set hper_target_embeddings_ratio as 0.9 and modify learning rate as 0.01 and longer training steps but failed still) 

Prompt: "A woman" 
Target prompt: "A woman with red lipsticks" 

Thanks

**Comment by HiPer0** [comment]
We recommend increasing the number of tokens from 5 to 10, as we have optimized 10 tokens for better human personalization. With the increased number of tokens, we observed better performance in preserving identities.

**Comment by waquey** [comment]
> We recommend increasing the number of tokens from 5 to 10, as we have optimized 10 tokens for better human personalization. With the increased number of tokens, we observed better performance in preserving identities.

Is there any suggested parameters such as learning rate, training steps, hper_target_embeddings_ratio for face conditions?
